browser-test/src/end_to_end_enumerators.test.ts

```diff
@@ -281,12 +281,10 @@ describe('End to end enumerator test', () => {
     await waitForPageJsLoad(page)
 
     await applicantQuestions.deleteEnumeratorEntity('Bugs')
-    await applicantQuestions.deleteEnumeratorEntity('Daffy')
```

Comment:
I had to update this test because the behavior is slightly different for deleting required enumerators now.

Before:
- **required non-enumerators**: Answer required question and save. Go back and try to clear required question and continue. Fails with "This question is required". Original data is **still there** and can be seen in review page
- **required enumerators**: Add enumerators and save. Go back and try to clear out all enumerators and continue. Fails with "This question is required". Original data is **removed** and no longer seen on review page.

After:
- **required non-enumerators**: no change in behavior (same as above)
- **required enumerators**: behavior now matches non-enumerators. Original data is still there and can be seen in review page when trying to remove required enumerator answers.

This change seems fine to me since we're unifying the behavior.

-------

Note: I actually cannot figure out why this is happening though. I think there might be something in [here](https://sourcegraph.com/github.com/civiform/civiform@3314e0b8debb78803bd09ee7d75cdee1d5658790/-/blob/server/app/services/applicant/ApplicantService.java?L231-233) or [here](https://sourcegraph.com/github.com/civiform/civiform@3314e0b8debb78803bd09ee7d75cdee1d5658790/-/blob/server/app/services/applicant/ApplicantService.java?L243) that results in us treating enumerators differently?